tests/driver/napi/nodejs.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use std::error::Error;
+use std::{fs::File, io::Write, path::PathBuf};
+
+lazy_static::lazy_static! {
+    static ref NODE_API_JS_PATH: PathBuf = {
+        let  node_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("../../../api/napi");
+
+        // On Windows npm is 'npm.cmd', which Rust's process::Command doesn't look for as extension, because
+         // it tries to emulate CreateProcess.
+         let npm = which::which("npm").unwrap();
+
+        // Ensure TypeScript is installed
+       std::process::Command::new(npm.clone())
+            .arg("install")
+            .arg("--no-audit")
+            .current_dir(node_dir.clone())
+            .stdout(std::process::Stdio::piped())
+            .stderr(std::process::Stdio::piped())
+            .output()
+            .map_err(|err| format!("Could not launch npm install: {}", err)).unwrap();
+
+        // Build the .js file of the NodeJS API from the .ts file
```

Comment:
Why is this step needed for the napi port?

Reply:
I'm asking because the previous step runs "npm install", which in `package.json` runs `npm build`. So it seems unnecessary to run `npm build` here again.

Why did the tests not work without it?

Reply:
Ok after a clean it works now.